cognite/powerops/resync/models/v2/powerops_models.py

```diff
@@ -201,9 +204,14 @@ def validate(cls, schema: Schema):
             referred_spaces[view.space].append(ref_view_id)
             if isinstance(view.filter, filters.Equals):
                 dumped = view.filter.dump()["equals"]["value"]
-                if "space" in dumped and "externalId" in dumped:
-                    node_id = NodeId(dumped["space"], dumped["externalId"])
-                    referred_node_types[node_id].append(ref_view_id)
+                try:
+                    if "space" in dumped and "externalId" in dumped:
+                        node_id = NodeId(dumped["space"], dumped["externalId"])
+                        referred_node_types[node_id].append(ref_view_id)
+                except Exception as exc:
```

Comment:
What was the failure that caused this one?

Reply:
This is just a guard for situations with unexpected `filter` values in a view, because the original error message was very cryptic. Specifically, confusion with `values` when using `in` vs. `value` when using `equals`.